app/src/main/java/com/example/parkingsystem/util/Constants.kt

```diff
@@ -0,0 +1,5 @@
+package com.example.parkingsystem.util
+
+object Constants {
+    const val parkingSize=10
```

Comment:
the constants should be in upper case and the words separated with _

should be PARKING_SIZE_DEFAULT

Reply:
Done 👍